src/ASM/ASMbase.C

```diff
@@ -235,6 +246,15 @@ int ASMbase::getNodeID (size_t inod, bool) const
 }
 
 
+char ASMbase::getLMType(size_t n) const
+{
+  if (n >= myLMs.first && n <= myLMs.second)
+    return myLMTypes[n-myLMs.first];
+
+  return 'L';
+}
```

Comment:
ok. har fått kommentarer på kall av interne metoder istedet for direkte access før så prøvde å preempte. fikset i aa533cf4d64521eb98db2d5107dc591e1d7fc4b4